apps/frontend/src/components/cards/EvaluationInfo.vue

```diff
@@ -61,5 +63,9 @@ export default class EvaluationInfo extends Vue {
       return e.id === this.file_object.database_id?.toString()
     })
   }
+
+  get start_time(): string | null {
+    return get_eval_start_time(this.file)
```

Comment:
If this is `return get_eval_start_time(this.file as ContextualizedEvaluation)` does that allow us to maintain typing on `get_eval_start_time`?